### PR TITLE
GUI: Back button returns to welcome window instead of application exit

### PR DESCRIPTION
**Pull request by djwfdev**
On both individual and batch processing screens, the back button navigates back to welcome window instead of terminating the whole application.

### CODE REVIEW

**Review by nwilde — APPROVED** [review]
Functionality for going back to the first page works well, don't know how to test the force link but looks good

File: src/Model/ForceLink.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        directory = os.listdir(file_path)
+    except FileNotFoundError:
+        logging.debug("Force link aborted, file not found")
+        return -1
+    except TypeError:
+        logging.debug("Force link aborted, file input invalid")
+        return -1
+    except NotADirectoryError:
+        logging.debug("Force link aborted, file input invalid")
+        return -1
```
Comment (pedrooste) [inline src/Model/ForceLink.py:47]: error debug level?

**Review by pedrooste — COMMENTED** [review]
Looks good, do we need to change the logging level?

Pretty hard to get a full idea / manually test code as its on a different fork/branch. Could be a limitation for all of our setups?

**Review by pedrooste — CHANGES_REQUESTED** [review]
Found how to test! In pycharm there is a PR tab you can access from the main Onko Project. Thanks @nwilde 

There is a problem when spamming between the "Individual Patient Curation" and main welcome page. Eventually leads to a segmentation fault, which can differ on your end. This was probably not originally introduced in this PR, but should get looked at?

Not completely sure what the Force Link does, but when I select nothing it seems to not do anything. I would suggest it has an initial disabled state if this is the case

**Pull request closed without merging**